### ISSUE TITLE
ASF-ui doesn't show changelog

### ISSUE THREAD

**Issue by Botan626** [JustArchiNET/ASF-ui#1070]
ui Releases page:
![image](https://user-images.githubusercontent.com/28785698/89537322-d9de1a80-d801-11ea-9a43-81ed10ef6406.png)

GitHub release page at the same time:
![image](https://user-images.githubusercontent.com/28785698/89538788-aac8a880-d803-11ea-98b4-5089153f0ff9.png)

**Comment by MrBurrBurr** [comment]
Paste your `ASF.json` please.

**Comment by Botan626** [comment]
{
  "Blacklist": [
    362890
  ],
  "ConfirmationsLimiterDelay": 15,
  "CurrentCulture": "en",
  "FarmingDelay": 10,
  "GiftsLimiterDelay": 3,
  "IdleFarmingPeriod": 4,
  "InventoryLimiterDelay": 15,
  "IPC": true,
  "LoginLimiterDelay": 15,
  "Statistics": false,
  "UpdateChannel": 2,
  "UpdatePeriod": 0,
  "SteamOwnerID": 76561197960895331,
  "SteamTokenDumperPluginEnabled": true
}

**Comment by MrBurrBurr** [comment]
Can't reproduce, works for me both with clean install and my own ASF.

**Comment by Botan626** [comment]
It did show changelog 1 hour after new ASF version was released.

Is it intended to be this way?

**Comment by MrBurrBurr** [comment]
Did clearing your cache fix the issue you had?

If the cached version is older than a day it will refetch it.

**Comment by Botan626** [comment]
Of course I did clear cache before, that's what I do al least one time after each update.

In my theory the line which indicates release time (e.g. 'Released 5 hours, 55 minutes ago') only starts to recognize release time when GitHub says new version was released at least 1 hour ago, which it can actually say after new version was e.g. 50 mins ago, as it rounds up this number.

I will check this with the new ASF release if I won't miss the 1st hour after release.
